.github/workflows/update-desktop-mirror.yml

```diff
@@ -0,0 +1,20 @@
+name: Build
```

Comment:
This file contains the "push-to-mirror" magic. On each merge to trunk, the contents of `desktop/Automattic/wp-desktop` are pushed to the `wp-desktop` repository (if there's a change to said contents).